Dockerfile-test

```diff
@@ -7,7 +7,7 @@ COPY ./ $HOME/
 RUN apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install -y curl git
 
-RUN export NODEV='20.12.2' \
```

Comment:
@nathanielrindlaub needed to upgrade for the docker tests to run. Merging for now but open to discussion